Formula/tensorflow.rb

```diff
@@ -0,0 +1,52 @@
+class Tensorflow < Formula
+  desc "Machine Intelligence Library"
+  homepage "https://tensorflow.org"
+  url "https://github.com/tensorflow/tensorflow.git",
+    :tag => "v0.7.1",
+    :revision => "028d0b46004c921acd48fdd0ec18128d79e18bf4",
+    :shallow => false
+
+  env :std
```

Comment:
how do I do that? I tried brew install --verbose --debug --list-checks tensorflow > buildlog.txt 2>&1 and it just hung